3_visualize.yml

```diff
@@ -24,3 +24,9 @@ targets:
       sites_sf = gw_sites_sf,
       svg_width = svg_width, 
       svg_height = svg_height)
+      
+  3_visualize/out/svg_list.csv:
+    command: list_svg_files(
+      target_name, 
+      I("src/assets"))
```

Comment:
I like the cleverness of creating a `list_svg_files()` function to pattern match and find svg files, but with how `scipiper` works, this target will only build once and will not update even if new SVGs are added to `src/assets`. I think it would be better to set it up, where you pass in the files that you want to make it into your list explicitly.

Reply:
^ though, in trying to figure out what to do, I realized that the filenames are created dynamically. This is making me wonder if we need a task table with dates instead ...

Reply:
What if there was an added input here that uses the current time period dates from the congif file?

Reply:
Removed this process for now. Similar to below, this seems like it will depend on how we store and access data in the jenkins build, and can be explored when that is resolved.